src/Microsoft.DotNet.Build.Tasks.Feed/BlobFeedAction.cs

```diff
@@ -22,28 +23,25 @@ sealed class BlobFeedAction
         private static readonly CancellationToken CancellationToken = TokenSource.Token;
 
         public BlobFeed feed;
-        public int MaxClients { get; set; } = 8;
-        
-        const string feedRegex = @"(?<feedurl>https:\/\/(?<accountname>[^\.-]+)(?<domain>[^\/]*)\/((?<token>[a-zA-Z0-9+\/]*?\/\d{4}-\d{2}-\d{2})\/)?(?<containername>[^\/]+)\/(?<relativepath>.*)\/)index\.json";
+        public string feedUrl;
 
-        public BlobFeedAction(string expectedFeedUrl, string accountKey, string indexDirectory, MSBuild.TaskLoggingHelper Log)
+        const string feedRegex = @"(?<feedurl>https:\/\/(?<accountname>[^\.-]+)(?<domain>[^\/]*)\/((?<token>[a-zA-Z0-9+\/]*?\/\d{4}-\d{2}-\d{2})\/)?(?<containername>[^\/]+)\/\/)index\.json";
+
+        public BlobFeedAction(string expectedFeedUrl, string accountKey, ITaskItem[] itemstoPush, MSBuild.TaskLoggingHelper Log)
         {
             this.Log = Log;
             Match m = Regex.Match(expectedFeedUrl, feedRegex);
-
             if (m.Success)
             {
                 string accountName = m.Groups["accountname"].Value;
                 string containerName = m.Groups["containername"].Value;
-                string relativePath = m.Groups["relativepath"].Value;
-                string feedUrl = m.Groups["feedurl"].Value;
```

Comment:
I expect we are still going to need the relativepath support for multiple feeds in one container and the feedurl support for our secure feeds.